pkg/controller/direct/securesourcemanager/mapper.generated.go

```diff
@@ -125,3 +201,39 @@ func SecureSourceManagerInstanceSpec_ToProto(mapCtx *direct.MapContext, in *krm.
 	}
 	return out
 }
+func SecureSourceManagerRepositorySpec_FromProto(mapCtx *direct.MapContext, in *pb.Repository) *krm.SecureSourceManagerRepositorySpec {
+	if in == nil {
+		return nil
+	}
+	out := &krm.SecureSourceManagerRepositorySpec{}
+	// MISSING: Name
+	// MISSING: Description
+	if in.GetInstance() != "" {
+		out.InstanceRef = &krm.SecureSourceManagerInstanceRef{External: in.GetInstance()}
+	}
+	// MISSING: Uid
+	// MISSING: CreateTime
+	// MISSING: UpdateTime
+	// MISSING: Etag
+	// MISSING: Uris
+	out.InitialConfig = Repository_InitialConfig_FromProto(mapCtx, in.GetInitialConfig())
+	return out
+}
+func SecureSourceManagerRepositorySpec_ToProto(mapCtx *direct.MapContext, in *krm.SecureSourceManagerRepositorySpec) *pb.Repository {
+	if in == nil {
+		return nil
+	}
+	out := &pb.Repository{}
+	// MISSING: Name
+	// MISSING: Description
+	if in.InstanceRef != nil {
+		out.Instance = in.InstanceRef.External
+	}
+	// MISSING: Uid
+	// MISSING: CreateTime
+	// MISSING: UpdateTime
+	// MISSING: Etag
```

Comment:
I guess Uid and etag are output only fields? If so we need to include them in observed state

Reply:
They both are, but they don't seem to be output in our API responses anyways. Would it be okay to leave it out and add it back later?

Reply:
Chatted with my team, unless we have a good reason to exclude the fields, i.e. Description, Etag, we should include all other fields in Beta resources.

Also, we need a fuzz test to test field coverage https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/3479289d515b28177576df38c20139f213f88b43/pkg/controller/direct/compute/forwardingrule/mapper.generated.go#L34, here's my fuzz [example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/3479289d515b28177576df38c20139f213f88b43/pkg/controller/direct/compute/forwardingrule/roundtrip_test.go), we need to leave a comment to explain the missing fields.

Reply:
Done, I copied the instance fuzz test

Reply:
editing the fuzz test again

Reply:
Done, I added one for the Observed State too